build.gradle.kts

```diff
@@ -75,6 +76,7 @@ dependencies {
     compileOnly("org.apache.logging.log4j:log4j-api:2.0-beta9")
     compileOnly("org.ow2.asm:asm-debug-all:5.0.3")
     compileOnly("org.apache.commons:commons-lang3:3.3.2")
+    compileOnly("org.jetbrains:annotations:23.0.0")
```

Comment:
unless this is included in kotlin include annotations in the versions buildscript

Reply:
when I was doing it it said reference not resolved; try "add org.jetbrains:annotations to the classpath"

Reply:
> when I was doing it it said reference not resolved; try "add org.jetbrains:annotations to the classpath"

cool